src-electron/db/db-mapping.js

```diff
@@ -142,6 +143,7 @@ exports.map = {
       description: x.DESCRIPTION,
       side: x.SIDE,
       isOptional: dbApi.fromDbBool(x.IS_OPTIONAL),
+      isFabricScoped: dbApi.fromDbBool(x.IS_FABRIC_SCOPED),
```

Comment:
Events do not have a concept of fabric-scoped.  They _do_ have a concept of fabric-sensitive.  And if we do this we can get rid of the `event_is_fabric_scoped` thing in helper-zcl.js, right?  Probably as a followup....

Reply:
Ok. I'll just rename this to IS_FABRIC_SENSITIVE.